moldesign/models/lammps.py

```diff
@@ -0,0 +1,370 @@
+        print 'Created LAMMPS data'
+
+        L.command("units real")
+        L.command("atom_style full")
+        L.command("pair_style lj/charmm/coul/long 8.0 10.0 10.0")
+        L.command("bond_style harmonic")
+        L.command("angle_style harmonic")
+        L.command("dihedral_style charmm")
+        L.command("improper_style harmonic")
+        L.command("kspace_style pppm 0.0001")
+        L.command("read_data " + dataPath)
+        
+        # NOTE: Do we want this?
+        L.command("neighbor " + str(self.params.skin) + " bin")
+        
+        L.command("thermo_style custom step temp pe etotal")
+        # NOTE: By default, time step is 10.0
+        L.command("thermo 10")
+        
+        # group hbonds
+        hbond_lj_types = _group_hbonds(parmedmol)
+        if len(hbond_lj_types) > 0:
+            L.command("group hbond type " + hbond_lj_types)
+            self.group_hbond = True
+        else:
+            self.group_hbond = False
+
+        # group water
+        water_res = _group_water(parmedmol)
+        if len(water_res) > 0:
+            L.command("group water molecule " + water_res)
+            self.group_water = True
+        else:
+            self.group_water = False
+
+        self.lammps_system = L
+        print 'Created LAMMPS object'
+    
+
+    def _group_hbonds(parmedmol):
+        hbond_group = ""
+        
+        if len(parmedmol.LJ_types) <= 0:
+            return hbond_groups
+
+        for nonbond_name, nonbond_idx in parmedmol.LJ_types.iteritems():
+            if(nonbond_name[0] == 'H'):
+                hbond_group = hbond_group + " " + str(nonbond_idx)
+        return hbond_group
+
+    def _group_water(parmedmol):
+        min_index = sys.maxint;
+        max_index = 0;
+        for res in parmedmol.residues:
+            if res.name == 'WAT':
+                if res.idx < min_index:
+                    min_index = res.idx
+                elif res.idx > max_index:
+                    max_index = res.idx
+        
+        water_group = ""
+        if(min_index != sys.maxint):
+            water_group = "<> {0} {1}" .format(min_index, max_index)
+
+        return water_group
+
+
+
+    # In order to simluate using LAMMPS, we need:
+    #   get molecule from pdb, cif, or name
+    #   assign forcefields
+    #   load parmed structure
+
+
+    # LAMMPS UNITS:
+    # For style real, these are the units:
+
+    # mass = grams/mole
+    # distance = Angstroms
+    # time = femtoseconds
+    # energy = Kcal/mole
+    # velocity = Angstroms/femtosecond
+    # force = Kcal/mole-Angstrom
+    # torque = Kcal/mole
+    # temperature = Kelvin
+    # pressure = atmospheres
+    # dynamic viscosity = Poise
+    # charge = multiple of electron charge (1.0 is a proton)
+    # dipole = charge*Angstroms
+    # electric field = volts/Angstrom
+    # density = gram/cm^dim
+
+    # See http://parmed.github.io/ParmEd/html/index.html for ParmEd units
+
+    def _create_lammps_data(parmedmol):
+        
```

Comment:
You'll of course need actually open a `lammps_data` file-like object here. It would be great if we could pass this file directly through the LAMMPS API so that we don't need to write anything to disk, but that's not always possible unfortunately.